go.mod

```
@@ -1,30 +1,28 @@
 module github.com/gardener/terraformer
 
-go 1.15
+go 1.16
 
 require (
 	github.com/aws/aws-sdk-go v1.35.26
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/gardener/gardener v1.12.4
-	github.com/go-logr/logr v0.1.0
+	github.com/gardener/gardener v1.17.0
```

You need https://github.com/gardener/gardener/pull/3611
```suggestion
	github.com/gardener/gardener v1.17.1
```

but it is still not released.